ledger/src/ui/test/bootloader/test_bootloader.c

```diff
@@ -190,7 +190,8 @@ void test_seed() {
     reset_flags();
     G_bootloader_mode = BOOTLOADER_MODE_DEFAULT;
     G_host_seed_is_set = false;
-    rx = set_apdu("\x80\x44"); // RSK_SEED_CMD
+    set_apdu("\x80\x44", 2); // RSK_SEED_CMD
```

Comment:
Maybe you can define a macro to call this function with a string literal using `sizeof()-1`? Same for the assertion one. It should only be used with string literals tho, so would try to make that clear if you do. You could also add `rx` as a parameter and set it within the macro.